product_search_category_attribute/models/product_search_category_attribute.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2024 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProductSearchCatAttr(models.AbstractModel):
+    _name = "product.search.cat.attr"
+    _description = "Product Search Category Attribute"
+
+    RELOAD_VIEW = ""
```

Comment:
this reload_view why is a general field of the model?

Reply:
Refer to def reload_view(self) method. It is a class attribute that needs to be defined in all the classes that inherit this one.